NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -242,35 +242,39 @@ def run(
             noise_temperature = np.power(10, temperature_interpolator(freqs[passband_filter]))
```

Comment:
This and the lines before: Why not caching the `scipy.interpolate.interp1d` functions

Reply:
Yes we discussed this.